lib/cli.js

```diff
@@ -24,6 +25,8 @@ updateNotifier({ pkg }).notify();
 
 if (argv.v) {
   console.log(`v${pkg.version}`);
+} else if (!isGit()) {
```

Comment:
I don't know, but we should keep the same structure in the entire project. I think if the stmt fits in one line, we should omit the braces.

In this issue they are more talking about to omit the braces or did I not get it right?